<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TelaPrincipal</class>
 <widget class="QMainWindow" name="TelaPrincipal">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TelaPrincipal</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="Buy_Button">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>530</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Comprar</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Hit_Button">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>530</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Bater</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Play_Button">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Play</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Reset_Button">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>10</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Reset</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Bet10_Button">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>530</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Bet 10</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Bet100_Button">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>530</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Bet 100</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Bet1000_Button">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>530</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Bet 1000</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Close_Button">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>10</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Fechar Jogo</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_player1">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>320</y>
      <width>91</width>
      <height>171</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_player2">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>320</y>
      <width>91</width>
      <height>171</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_player3">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>320</y>
      <width>91</width>
      <height>171</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_dealer3">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>100</y>
      <width>91</width>
      <height>171</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_player5">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>320</y>
      <width>91</width>
      <height>171</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_dealer4">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>100</y>
      <width>91</width>
      <height>171</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_dealer5">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>100</y>
      <width>91</width>
      <height>171</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_player4">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>320</y>
      <width>91</width>
      <height>171</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_dealer2">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>100</y>
      <width>91</width>
      <height>171</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_dealer1">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>100</y>
      <width>91</width>
      <height>171</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Next_Button">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>10</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Next</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="plainTextEdit_player">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>420</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="plainTextEdit_dealer">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>200</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="plainTextEdit_playerBank">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>350</y>
      <width>91</width>
      <height>61</height>
     </rect>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="plainTextEdit_dealerBank">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>91</width>
      <height>61</height>
     </rect>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="plainTextEdit_tablePot">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>10</y>
      <width>91</width>
      <height>61</height>
     </rect>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="plainTextEdit_Result">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>10</y>
      <width>271</width>
      <height>61</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
